src/acl/AclDenyInfoList.h

```diff
@@ -36,7 +35,7 @@ class AclDenyInfoList
     }
     err_type err_page_id = ERR_NONE;
     char *err_page_name = nullptr;
-    AclNameList *acl_list = nullptr;
+    SBufList acl_list;
```

Comment:
Please document what this member is, especially after we obscure its meaning by using a generic SBuf container. For example:

```suggestion
    SBufList acl_list; ///< ACL names in configured order
```